install/kubernetes/cilium/Chart.yaml

```diff
@@ -20,3 +20,34 @@ sources:
 links:
   - name: eBPF.io
     url: https://ebpf.io/
+annotations:
+  artifacthub.io/crds: |
```

Comment:
What about CiliumNode, CiliumIdentity, CiliumEndpoint?

Reply:
I didn't bother with those since they're managed directly by Cilium, and other than providing a way to introspect cluster state, users won't be interacting with them (unlike the ones that are here, which are explicitly configured by the user).

:shrug: maybe we should just list all of them.

Reply:
Yeah that makes sense. There is one specific workflow for CiliumNode where we ask the user in ENI mode if they want to change the interface offset (`first-interface-index`) for their ENI setup, to do it by modifying the CN resource. https://docs.cilium.io/en/v1.9/concepts/networking/ipam/eni/#ip-allocation

At the very least, we should "expose" CN. But yeah, I don't mind either way if we include all or keep it as is + CN.